tooling/reproducible/ReproducibleBuilds.md

```diff
@@ -79,17 +83,55 @@ before the comparable_patch.sh can be run.
 - For BinRepl.class : export CLASSPATH=<temurin-build>/tooling/src/java:$CLASSPATH
 - A JDK for running BinRepl java : export PATH=<jdk>/bin:$PATH
 
+##### Cygwin treacherousness
+
+- It is extremely difficult (maybe impossible) to invoke `vcvarsall.bat+cl` in cygwin directly
+- Thus, it is recommended to launch this via `cmd -c` or preferably by an executable `.bat` file such as:
+
+```bash
+ pushd "$MSVSC/BUILD/TOOLS"
+    rm -f WindowsUpdateVsVersionInfo.obj
+    echo "
+      call vcvars64.bat
+      cl $(cygpath -m $YOUR_WORKDIR/temurin-build/tooling/src/c/WindowsUpdateVsVersionInfo.c) version.lib
+    " > bat.bat
```

Comment:
I would give the bat file a useful name

Reply:
Imagination betrayed me.  `setupEnvAndCompile.bat` ?

Reply:
I liked this partially nonsense name. It is hack around the need to `call  *vars*``  and is very short living. The non perfect name is highlighting that.

Anywway, renamed to ` setupEnvAndCompile.bat` for now.  Feel free to poke if you still would like to improve it.